### ISSUE TITLE
Spilt the MySQL database script

### ISSUE THREAD

**Issue by StarOnTheSky** [MrCosmo666/Teeworlds-MRPG#146]
Currently all data is stored in the MySQL database, including player info, dialog info, item info, etc. I find it troublesome to update the database when I try to update the game server.  Can we split the database script into two parts? The first one contains player info, which is updated using `modify table`; and another script contains game info (dialog, item), updated by `drop` and `create`.

**Comment by kurosio** [comment]
@StarOnTheSky, Of course, the database is large and it is still being optimized, but only in conjunction with json sql
